roles/sysbench/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
Want to either delete or update this boilerplate stuff?

Reply:
Done.